starter-dev-e2e/cypress/fixtures/profile/graphql/empty/userProfile.empty.graphql.json

```diff
@@ -0,0 +1,32 @@
+{
+  "data": {
+    "user": {
+      "id": "MDEyOk9yZ2FuaXphdGlvbjIyODM5Mzk2",
```

Comment:
I already updated the old fixture for this in my PR, but we can probably anonymize this -- I just use `TESTUSERID`